Adding a computed attribute for the repository owner to the github_repository (#2675)
integrations/terraform-provider-github: website/docs/d/repository.html.markdown

* `full_name` - (Optional) Full name of the repository (in `org/name` format).

> * `owner` - (Optional) Owner of the repository. If not provided, the owner specified in the provider configuration will be used.

Reviewer: ```suggestion
* `owner` - (Optional)  Owner of the repository. If not provided, the provider's default owner is used.
```

This is the phrase used in other resources, since no owner is specified in the provider's configuration.
